code/anim/animplay.h

```diff
@@ -59,7 +58,7 @@ int				anim_stop_playing(anim_instance* anim_instance);
 int				anim_show_next_frame(anim_instance *instance, float frametime);
 void				anim_release_all_instances(int screen_id = 0);
 void				anim_release_render_instance(anim_instance* instance);
-anim			  *anim_load(const char *name, int cf_dir_type = CF_TYPE_ANY, int file_mapped = PAGE_FROM_MEM);
+anim			  *anim_load(const char *name, int cf_dir_type = CF_TYPE_ANY, int file_mapped = PAGE_FROM_DISK);
```

Comment:
The generic animation system uses the integer literal `0` for the `file_mapped` parameter which now causes a change of behavior. That should be fixed to use the appropriate enum value. The code for this is in [generic.cpp, line 196](https://github.com/scp-fs2open/fs2open.github.com/blob/master/code/graphics/generic.cpp?utf8=%E2%9C%93#L196).

Everything else uses the default parameter value so there should be no change needed.